genai-perf/genai_perf/export_data/csv_exporter.py

```diff
@@ -100,6 +128,55 @@ def _write_system_metrics(self, csv_writer) -> None:
             value = self._stats[metric.name]["avg"]
             csv_writer.writerow([metric_str, f"{value:.2f}"])
 
+    def _write_telemetry_aggregated_metrics(self, csv_writer) -> None:
```

Comment:
The breaks in the csv we already had in place cause a lot of problems for downstream consumers.
Ideally, they would use the json, but I would update this to use the existing tables as much as possible.